### PR TITLE
Bug 1068973 - Groups headers above, and footers below thumbnails.

### PR DESCRIPTION
**Pull request by eeejay**

### CODE REVIEW

File: apps/gallery/test/marionette/lib/gallery.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ Gallery.ORIGIN = 'app://gallery.gaiamobile.org';
 Gallery.Selector = Object.freeze({
   thumbnail: '.thumbnail',
   fullscreenView: '#fullscreen-view',
-  thumbnailsView: '#thumbnail-list-view',
+  thumbnailsView: '#thumbnail-footers > .thumbnails-list',
```
Comment (punamdahiya) [inline apps/gallery/test/marionette/lib/gallery.js]: I think this should be body.thumbnailListView #thumbnail-headers

File: tests/python/gaia-ui-tests/gaiatest/apps/gallery/app.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ class Gallery(Base):
     _empty_gallery_title_locator = (By.ID, 'overlay-title')
     _empty_gallery_text_locator = (By.ID, 'overlay-text')
     _progress_bar_locator = (By.ID, 'progress')
-    _thumbnail_list_view_locator = (By.ID, 'thumbnail-list-view')
+    _thumbnail_list_view_locator = (By.CSS_SELECTOR, '#thumbnail-footers > .thumbnails-list')
```
Comment (punamdahiya) [inline tests/python/gaia-ui-tests/gaiatest/apps/gallery/app.py]: Same here: body.thumbnailListView #thumbnail-headers

File: apps/gallery/test/marionette/lib/gallery.js (hunk 1 above)
Comment (punamdahiya) [inline apps/gallery/test/marionette/lib/gallery.js]: I take it back, after looking at the test thumbnailsView name is confusing and should be updated to thumbnailListFooter

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=958587fbb910)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c8732835a6f9)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=46bc487fafc2)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=24e835bd0f45)

File: apps/gallery/index.html [hunk 3]
```diff
@@ -59,26 +59,32 @@
   </head>
 
   <body role="application" class="theme-media">
-    <section role="region" id="thumbnail-list-view" class="skin-dark">
-      <!-- Header only shown on tablet view  -->
-      <gaia-header id="thumbnails-list-top">
+    <section role="region" id="thumbnail-views" class="skin-dark">
```
Comment (davidflanagan) [inline apps/gallery/index.html]: I'd appreciate a comment above this line saying something like:

  This section holds the thumbnails list and the headers and footers for all views that display the thumbnails

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3759ec549e4e)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=750fa083bf9e)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=118dca0fc4da)

**Pull request merged**